.github/workflows/node-tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Run Node Unit Tests
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macos-12
+          - ubuntu-22.04
+          - windows-2022
+        node-version:
+          - '14'
+          - '15'
+          - '16'
+          - '17'
+          - '18'
+          - '19'
```

Comment:
We can probably drop v15 and v17 as those aren't LTS releases and are already EOL